sigma/backends/elasticsearch/__init__.py

```diff
@@ -1,5 +1,7 @@
-from .elasticsearch import LuceneBackend
+from .elasticsearch_lucene import LuceneBackend
+from .elasticsearch_eql import EqlBackend
 
 backends = {
```

Comment:
This is not really required anymore with pySigma 0.10 autodiscovery feature.

Reply:
But at least for the local tests which don't participate in the autodiscovery. 
Am I wrong?

Reply:
For these you should instantiate the backend directly.